### DISCUSSION TITLE
how to run a yolo project on google coral accelerator

### DISCUSSION THREAD

**Discussion by StefanoGarzoni** [roboflow/notebooks#153] [Q&A]
hello everyone, I bought a google coral accelerator a few days ago, it seemed like a valid option for not spending hundreds of euros for a new gpu, but only now that I have it in my hands do I notice the difficulties in using it!
I'm on a windows device, and I wanted (after installing it by following the specific guide : https://coral.ai/docs/accelerator/get-started/#next-steps) to test it on some project with yolo that I already own (I'm the same ones you posted on youtube), but unfortunately it's not intuitive at all...
There are no guides on the net to try it with a yolo project on windows or from anaconda3 and I was wondering if anyone could give me a hand..
As usual, thank you in advance for the help you always give me and for your time.
I hope someone can help me solve this problem!

**Comment by SkalskiP** [comment]
Hi, @Stefano0513 👋🏻 sorry to hear that. Unfortunately, I have no experience with google coral accelerator. Maybe other users from community will be able to help you.

**Reply by StefanoGarzoni** [reply]
May I ask if you've ever had to deal with other accelerators like the nvidia or devboard ones?

**Reply by SkalskiP** [reply]
Hi 👋🏻 I have quite a lot of experience with NVIDIA Jetson.

**Reply by StefanoGarzoni** [reply]
ok thanks, then maybe i'll go for a jetson
